tasks/jitsi-meet.yml

```diff
@@ -1,15 +1,14 @@
 ---
 
-- name: Fetch signing key for Jitsi repository
-  apt_key:
-    id: "66A9CD0595D6AFA247290D3BEF8B479E2DC1389C"
-    url: "https://download.jitsi.org/jitsi-key.gpg.key"
-    state: present
+- name: Copy Jitsi repository key
```

Comment:
You can use the `apt_key` module from ansible with the parameter `file` to ensure the key is in the keyring

Reply:
Using the built-in keyring is not the recommended method for adding external repository anymore, though, because that would allow those external keys to sign/fake packages from the official debian repositories.